### DISCUSSION TITLE
analogWrite() not working

### DISCUSSION THREAD

**Discussion by kaichiao** [espressif/arduino-esp32#9482] [General]
Hi, there.

I found a bug from core 2.0.9 to now.
in the code, if remark Serial.println(), analogWrite() will not working.

Thanks for reading.

`
const byte MotorA1 = 32;
const byte MotorA2 = 33;

void setup() {
  pinMode(MotorA1,OUTPUT);
  pinMode(MotorA2,OUTPUT);
  Serial.begin(9600);
}

void loop() {
  //Serial.println();
  analogWrite(MotorA1, 127);  //255 duty cycle 100%
  analogWrite(MotorA2, 0);
}
`

**Comment by me-no-dev** [comment]
Not a bug. You are resetting the PWM too often. Call once you need to change the value, not in a loop
